app/models/cms/email_template.rb

```diff
@@ -23,24 +23,6 @@ def self.reset_templates_path!
     service_contract_messenger_expired_trial_period_notification
   ].freeze
 
-  PROVIDER_TEMPLATES = %w[
-    account_messenger_expired_credit_card_notification_for_provider
-    alert_messenger_limit_alert_for_provider
-    alert_messenger_limit_violation_for_provider
-    invoice_messenger_unsuccessfully_charged_for_provider
-    invoice_messenger_unsuccessfully_charged_for_provider_final
-    account_messenger_invoices_to_review
-    account_messenger_new_signup
-    account_messenger_plan_change_request
-    cinstance_messenger_contract_cancellation
-    cinstance_messenger_new_application
-    cinstance_messenger_plan_change
-    service_contract_messenger_contract_cancellation
-    service_contract_messenger_new_contract
-    service_contract_messenger_plan_change
-    data_export
-  ].freeze
```

Comment:
Do we do anything to remove existing templates? Or will they be removed only after the accounts are removed?

Reply:
This PR doesn't include removing templates or dispatch rules, so option 2.

Reply:
I'd suggest adding a JIRA to remove `MailDispatchRule` class and its table. For email_templates idk

Reply:
I created the Jira for mail dispatch rules: https://issues.redhat.com/browse/THREESCALE-11825

About templates, I don't think its needed. Only buyer templates can be overridden by custom templates stored in DB. Provider templates are just files, can't be customized, and I'm already deleting all of them in this PR. I mean, templates from the old system.